syntax = "proto2";

package apollo_fuel.fueling.control.proto.feature_key;

message FeatureKey {
    optional double speed_step = 1;
    optional double acc_step = 2;
    optional double steer_step = 3;
    optional double speed_max = 4;
    optional double throttle_max = 5;
    optional double brake_max = 6;
    optional string vehicle_type = 7;
}
